crates/bevy_input/src/gamepad.rs

```diff
@@ -1666,10 +1715,10 @@ mod tests {
             (0.99, None, Some(1.0)),
             (0.96, None, Some(1.0)),
             (0.95, None, Some(1.0)),
-            (0.9499, None, Some(0.9499)),
-            (0.84, None, Some(0.84)),
-            (0.43, None, Some(0.43)),
-            (0.05001, None, Some(0.05001)),
+            (0.9499, None, Some(0.9998889)),
```

Comment:
These lines need a comment explaining why they don't match the obvious value.

Reply:
Hmm, this is a recurring problem 🤔

Reply:
I copied it from the PR that I adopted. I'll add a comment once I can compile and test bevy again.